ui/components/multichain/activity-list-item/activity-list-item.js

```diff
@@ -66,7 +66,11 @@ export const ActivityListItem = ({
         gap={4}
       >
         {icon && <Box display={Display.InlineFlex}>{icon}</Box>}
-        <Box display={Display.InlineFlex} width={BlockSize.Full}>
+        <Box
+          display={Display.InlineFlex}
+          width={BlockSize.Full}
+          style={{ overflow: 'hidden' }}
```

Comment:
I know it's a little more overhead on the engineering side but I'm hesitant to recommend inline styles because I think classNames are in general more scalable, performant and have a better developer experience.

Reply:
added in db62a525f0884a768bd2e5d066241b7cfcf6f10d